### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rohitsangwan01**
Added few projects

PocketBase framework compiled into mobile framework : 
https://github.com/rohitsangwan01/pocketbase_mobile

A Flutter app to directly start PocketBase server from mobile
https://github.com/rohitsangwan01/pocketbase_server_flutter

A PocketBase server to start directly from native android
https://github.com/rohitsangwan01/pocketbase_server_android_example

A chat app built using PocketBase and Flutter
https://github.com/rohitsangwan01/flutter_pocketbase_chat

i have a complete blog about these projects 
https://rohitsangwan.hashnode.dev/run-pocketbase-from-mobile

### CODE REVIEW

**Comment by benallfree** [comment]
@rohitsangwan01 This looks awesome, can you please correct `pocketbase` to `PocketBase`? Just for consistency :)

**Comment by benallfree** [comment]
Also, can you explain the meaning of the term `proper`? What is a proper chat app?

I'm curious too, did you really mean PocketBase can be compiled and run on iOS and Android? How?

**Comment by rohitsangwan01** [comment]
> Also, can you explain the meaning of the term `proper`? What is a proper chat app?

By proper chatApp i meant , a chat app with Realtime Chat updates, sharing images , audio messages and all using PocketBase only, and yes with the help of my PocketBaseMobile project, we can use this chatApp which will have a server within the mobile ,and other users will be able to communicate by connecting under same network 

> I'm curious too, did you really mean PocketBase can be compiled and run on iOS and Android? How?

Yes pocketBase can be compiled on IOS and Android using PocketBase go framework and GoMobile , i also mentioned the repo's in which i actually compiled pocketBase in both android and IOS , and for ui consistency am using flutter there ( but can be used natively too )

also here is a demo app to run pocketBase directly from android
https://play.google.com/store/apps/details?id=com.pocketbase.mobile&hl=en_US

**Comment by rohitsangwan01** [comment]
> 1. What is the use case for running a PocketBase server from a mobile device?

Well there can be multiple use cases like , to create a secure localChat server or maybe a file storage system or anything which provided by pocketBase, 
To use an old android device as local server to host pocketBase , someone mentioned it [here](https://github.com/pocketbase/pocketbase/issues/1768)
To use a mobile app which is based on pocketBase, so we can simply run pocketBase server within mobile and show demo of our application ( or for personal use as well ) , useful for someone who don't want to host pocketBase yet
Or maybe a students team want to show demo of a mobile app but can't afford hosting or something , so all they have to do is , run pocketBase server from there mobile and their main app which is based on pocketBase is ready to use
Am sure there might be other useCases as well


> 2. You have built a mobile chat client, do you have a corresponding project for the chat server?

a project for chat server not required, all we have to do is run pocketBase, and follow these few [steps](https://github.com/rohitsangwan01/flutter_pocketbase_chat#getting-started) to setup pocketBase, and that's it

**Comment by benallfree** [comment]
Okay I understand everything now. Yes, this is awesome! :) 

Some ideas to make your projects clearer for others (like me):

* Add those use cases to the documentation. I think they are all interesting but I had not thought of any of them myself.
* Yours is the first I've seen where a custom schema is required to create a compatible pocketbase backend. This is an interesting idea and maybe we could collaborate with PocketHost. Right now, PocketHost can create empty instances, but it would be cool to have "one click apps" where it creates an instance with a ready-made schema. So for example, the user could create a `PocketChat` instance which would install the required schema. Then, they have an instant chat backend compatible with your chat frontend. What do you think?

**Comment by benallfree** [comment]
Please capitalize `PocketBase` and `Flutter` and then I think this is ready to merge.

**Comment by rohitsangwan01** [comment]
@benallfree  Yes i think that would be better to add usecases in docs

And also this seems to be a cool idea , where we can kind of host our own kind of images of pocketBase on pocketHost like you said pocketChat and users can simply install that preConfigured image
Am not sure if we can create schema using any Api's of pocketBase , if yes then one of the option will be to simply add an option to initialise or something in projects which will automatically setup required things in pocketBase

File: README.md [hunk 1]
```diff
@@ -70,6 +71,7 @@ This is a collection of [PocketBase](https://pocketbase.io) community resources.
 ## Other tools
 
 * [PocketBaseUML](https://pocketbase-uml.github.io/) - A free, open-source web application that generates UML diagrams based on PocketBase databases.
+* [PocketBaseMobile](https://github.com/rohitsangwan01/pocketbase_mobile) - Android and IOS frameworks for running pocketbase from mobile
```
Comment (benallfree) [inline README.md:74]: PocketBase
Flutter

**Review by benallfree — CHANGES_REQUESTED** [review]

**Pull request merged**